sdk/eventhub/event-hubs/src/eventHubSender.ts

```diff
@@ -280,7 +280,7 @@ export class EventHubSender extends LinkEntity {
             operation: () => this._init(options),
             connectionId: this._context.connectionId,
             operationType: RetryOperationType.senderLink,
-            times: Constants.defaultConnectionRetryAttempts,
+            times: Constants.defaultMaxRetriesForConnection,
```

Comment:
Update `times` -> `maxRetries` in retry config before passing it to `retry` method.